src/controllers/browser.ts

```diff
@@ -63,6 +63,9 @@ export async function initWhatsapp(sessionId?: string, puppeteerConfigOverride?:
 export async function injectApi(page: Page) {
   // const preloadFile = fs.readFileSync('./preload', 'utf8');
   // await page.evaluateOnNewDocument(preloadFile);
+  await page.addScriptTag({
+    path: require.resolve(path.join(__dirname, '../lib', 'sha256.js'))
```

Comment:
If this is required for a feautre of WAPI.js itself to work then it should be a oneliner in there already.